src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -829,8 +829,20 @@ if (typeof Sfjs === 'undefined' || typeof Sfjs.loadToolbar === 'undefined') {
             initializeLogsTable: function() {
                 Sfjs.updateLogsTable();
 
-                document.querySelectorAll('.log-filter input').forEach((input) => {
-                    input.addEventListener('change', () => { Sfjs.updateLogsTable(); });
+                document.querySelectorAll('button[data-filter-name="type"]').forEach((tab) => {
+                    tab.addEventListener('click', () => {
+                        /* update the currently selected "log type" tab */
+                        document.querySelectorAll('button[data-filter-name="type"].tab-control').forEach((otherTab) => otherTab.classList.remove('active'));
+                        tab.classList.add('active');
+
+                        Sfjs.updateLogsTable();
+                    });
+                });
+
+                document.querySelectorAll('.log-filter-option input[type="checkbox"]').forEach((filterOption) => {
+                    filterOption.addEventListener('click', () => {
```

Comment:
this should use the `change` event (as done before), not the `click` event, to properly react to keyboard-based changes to the checkbox.